api/base/settings/defaults.py

```diff
@@ -51,21 +51,24 @@
 }
 
 REST_FRAMEWORK = {
-    'PAGE_SIZE': 10,
-
-    # Order is important here because of a bug in rest_framework_swagger. For now,
```

Comment:
I don't know. If it works and we don't have to change everyone else's code, sure. If not, I added comments to Steve's issue on django-rest-swagger and your issue on django-rest-framework-json-api to see about fixing it properly. Let's leave swagger broken as you have it right now if it can't be all correct, though.